include/fastdds/rtps/transport/UDPTransportDescriptor.h

```diff
@@ -31,28 +31,31 @@ namespace rtps{
  * - interfaceWhiteList: Lists the allowed interfaces.
  * @ingroup TRANSPORT_MODULE
  */
-typedef struct UDPTransportDescriptor: public SocketTransportDescriptor
+typedef struct UDPTransportDescriptor : public SocketTransportDescriptor
```

Comment:
Avoid typedef trick.
Check class documentation, as it seems to only mention fields on the parent class